erpnext/selling/doctype/quotation/quotation.json

```diff
@@ -5,10 +5,12 @@
  "autoname": "naming_series:", 
  "beta": 0, 
  "creation": "2013-05-24 19:29:08", 
- "custom": 0,
+ "custom": 0, 
+ "default_print_format": "test123", 
```

Comment:
I remember fixing this. https://github.com/frappe/erpnext/commit/39c952a3f2058a494bf12ee806f25db6bb28da74